framework/commercelayer/api/endpoints/checkout/checkout.ts

```diff
@@ -0,0 +1,36 @@
+import type { CheckoutEndpoint } from '.'
+import getCredentials from '@framework/api/utils/getCredentials'
```

Comment:
You shouldn't import from framework inside the provider, just relative, otherwise it may break other providers

Reply:
Alright, I'll fix that.